src/main/java/ch/njol/skript/effects/EffReplace.java

```diff
@@ -110,7 +126,12 @@ protected void execute(final Event e) {
 	
 	@Override
 	public String toString(final @Nullable Event e, final boolean debug) {
-		return "replace " + needles.toString(e, debug) + " in " + haystack.toString(e, debug) + " with " + replacement.toString(e, debug);
+		if (replaceFirst)
+			return "replace first " + needles.toString(e, debug) + " in " + haystack.toString(e, debug) + " with " + replacement.toString(e, debug)
+					+ "(case sensitive: " + caseSensitive + ")";
+		else

```

Comment:
Minor nitpick, but you don't need an else here, since the other case is already handled.

Reply:
Fixed :)